test/python/picongpu/compiling/runner.py

```diff
@@ -120,7 +120,7 @@ def test_build(self):
         # but: injecting a generated (but not built) dir
         # from another runner works
         other_runner = Runner(self.sim)
-        other_runner.generate()
+        other_runner.generate(printDirToConsole=True)
```

Comment:
Why is this changed? Could it be this is a left over from testing?

Reply:
I saw this in many other files too.

Reply:
That is a quality of live improvement for a maintainer using the compile tests.
If the developer made an error in the templates or schema only, the quick tests will pass, since they do not test the that the generated picInput is acutally valid for PIConGPU(compiles).
Only the compiling tests will fail, but they fail with a cryptic error message, along the lines of "internal process has failed", the debugSwitch makes it so that in addition to the error message the runner prints the temporary (and automatically) path it generates the picInput to before actually running `pic-build`.
On a fail you may then go to the temp directory and manually call `pic-build` yourself to see the actual PIConGPU compile error messages and figure out what went wrong.
Before you had no idea where to look without looking into the code nor any idea which of the several picInputs had actually generated the error.